CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+
+## [1.51](https://github.com/brave/brave-ios/releases/tag/v1.51)
+
+### Web3
+
+- Added network icons to be shown for native assets. ([#7123](https://github.com/brave/brave-ios/issues/7123))
+- Implemented Market tab in Brave Wallet. ([#7202](https://github.com/brave/brave-ios/issues/7202))
+- Implemented NFT tab in Brave Wallet. ([#7173](https://github.com/brave/brave-ios/issues/7173))
+- Updated label for "Reset and clear wallet data" under Wallet settings. ([#7337](https://github.com/brave/brave-ios/issues/7337))
```

Comment:
nit: think `Wallet` doesn't need to be capitalized in this instance unless you refer to `Brave Wallet`